packages/fxa-auth-server/lib/oauth/jwt.js

```diff
@@ -58,3 +60,14 @@ exports.verify = async function verify(jwt, options = {}) {
     ignoreExpiration: options.ignoreExpiration,
   });
 };
+
+function normalizeTyp(typ) {
+  // Ref https://tools.ietf.org/html/rfc7515#section-4.1.9 for the rules.
+  if (typ) {
+    typ = typ.toLowerCase();
+    if (typ.indexOf('/') === -1) {
```

Comment:
I suppose `includes` reads a little better here.

Reply:
Oh yes, thanks! My javascript is often stuck in the late 90s 😆